lifecycle/README.rst

```diff
@@ -1,11 +1,11 @@
 Introduction
 ------------
 
-ROS2 introduces the concept of managed nodes, also called ``LifecycleNode``\ s.
+ROS 2 introduces the concept of managed nodes, also called ``LifecycleNode``\ s.
 In the following tutorial, we explain the purpose of these nodes, what makes them different from regular nodes and how they comply to a lifecycle management.
 Managed nodes are scoped within a state machine of a finite amount of states.
 These states can be changed by invoking a transition id which indicates the succeeding consecutive state.
-The state machine is implemented as described at the `ROS2 design page <http://design.ros2.org/articles/node_lifecycle.html>`__.
+The state machine is implemented as described at the `ROS 2 design page <http://design.ros2.org/articles/node_lifecycle.html>`__.
 
 Our implementation differentiates between ``Primary States`` and ``Transition States``.
 Primary States are supposed to be steady states in which any node can do the respected task.
```

Comment:
I think this terminology is awkward. A state machine has states and transitions. I think it's potentially confusing to define a "transition state."

Reply:
While I agree with you in an abstract sense, we do indeed define "transition states" as part of the state machine: https://github.com/ros2/rcl_interfaces/blob/master/lifecycle_msgs/msg/State.msg .  So this sentence matches the code, even though the terminology is a bit weird.

Reply:
Ok. Good that the docs are in sync.